cropper/src/main/java/com/canhub/cropper/CropOverlayView.kt

```diff
@@ -34,6 +34,18 @@ class CropOverlayView
 
     companion object {
 
+        /**
+         * Creates the paint object for drawing text label over crop overlay
+         */
+        private fun getTextPaint(options: CropImageOptions): Paint {
+            return Paint().apply {
+                strokeWidth = 1f
+                textSize = options.cropperLabelTextSize
+                style = Paint.Style.FILL
+                textAlign = Paint.Align.CENTER
+                this.color = options.cropperLabelTextColor
+            }
+        }
```

Comment:
Please change to:
`/** Creates the paint object for drawing text label over crop overlay */`

`private fun getTextPaint(options: CropImageOptions): Paint = `, removing `return`